modules/covid_vaccine/app/jobs/covid_vaccine/expanded_registration_email_job.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+require 'va_notify/service'
+
+module CovidVaccine
+  class ExpandedRegistrationEmailJob
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options expires_in: 1.day, retry: 2
+
+    STATSD_ERROR_NAME = 'worker.covid_vaccine_expanded_registration_email.error'
+    STATSD_SUCCESS_NAME = 'worker.covid_vaccine_expanded_registration_email.success'
+
+    def perform(record_id)
+      submission = CovidVaccine::V0::ExpandedRegistrationSubmission.find(record_id)
+      return if submission.email_confirmation_id.present?
+
+      notify_client = VaNotify::Service.new(Settings.vanotify.services.va_gov.api_key)
+      template_id = Settings.vanotify.services.va_gov.template_id.covid_vaccine_registration
+      email_address = submission.raw_form_data[:email]
+
+      notify_response = notify_client.send_email(email_address: email_address, template_id: template_id,
+                                                 personalisation: { 'date' => formatted_date(submission.created_at),
+                                                                    'confirmation_id' => submission.submission_uuid },
```

Comment:
BTW, I'm not entirely sure if `confirmation_id` is the actual personalization key again, this was just an assumption I made.

Reply:
It is, I connected to VANotify staging and got errors when I omitted it.